stdlib/io.pyi

```diff
@@ -75,7 +75,10 @@ class IOBase(metaclass=abc.ABCMeta):
     def __del__(self) -> None: ...
     @property
     def closed(self) -> bool: ...
-    def _checkClosed(self, msg: str | None = ...) -> None: ...  # undocumented
+    if sys.version_info >= (3, 13):
+        def _checkClosed(self) -> None: ...  # undocumented
```

Comment:
I couldn't find any references for this change though

Reply:
I just checked against the source, and at least from Python 3.8, `_checkClosed()` didn't take any arguments. This seems also true at runtime:

```
Python 3.8.18 (default, Apr 29 2024, 19:58:11) 
[GCC 13.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from io import IOBase
>>> IOBase()._checkClosed()
>>> IOBase()._checkClosed("")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: _checkClosed() takes no arguments (1 given)
```

Could you remove the version check and just use your new version without the argument?

Reply:
That's interesting. I was looking at the C source code. Seems that there is some discrepancy there. But in any case it's better to use the tighter version.